nextflow_schema.json

```
@@ -326,6 +326,13 @@
                     "hidden": true,
                     "description": "BAMTools JSON file with custom filters for single-end data.",
                     "fa_icon": "fas fa-cog"
+                },
+                "shift_reads": {
+                    "type": "boolean",
+                    "fa_icon": "fas fa-chart-area",
+                    "default": false,
+                    "help_text": "Shift aligned reads as commonly done for ATACseq, +4bp for reads on the + strand, -5 bp for reads on the - strand. This can only be applied if all samples are paired-end.",
```

Usually, this is done for footprinting, should we add it to the `help_text`?
```suggestion
                    "help_text": "Shift aligned reads as commonly done for ATACseq footprinting analysis, +4 bp for reads on the + strand, -5 bp for reads on the - strand. This can only be applied if all samples are paired-end.",
```